src/runtime_src/core/pcie/tools/xbutil/xbutil.cpp

```diff
@@ -989,6 +989,44 @@ int xcldev::device::runTestCase(const std::string& py,
     return runShellCmd(cmd, output);
 }
 
+int xcldev::device::runXbTestCase(const std::string& test)
+{
+    struct stat st;
+    int retVal;
+    std::string output;
+
+    std::string devInfoPath = std::string(m_devinfo.mName) + "/test/";
```

Comment:
Please read the name from sysfs node:

```
    std::string name, errmsg;
    pcidev::get_dev(m_idx)->sysfs_get( "rom", "VBNV", errmsg, name );
    if (!errmsg.empty()) {
        std::cout << errmsg << std::endl;
        return -EINVAL;
    }

    std::string devInfoPath = name + "/test/";
```

Reply:
Fixed this, and another occurrance as well.